### PR TITLE
Fix: DO-3605 code viewer code area overflows container

### PR DESCRIPTION
**Pull request by patricia-causalens**
## Motivation and Context

Noticed that code viewer component could overflow its container if width was set to it

## Implementation Description

Fixed so that it can scroll horizontally when needed.

Also when updating the changelogs noticed that all of the ui packages did not have their NEXT replaced. When looking into it this was due to the changelog being placed one layer deeper than other dara packages. I moved them so that these are all the same now.

## Any new dependencies Introduced

None

## How Has This Been Tested?

Locally on storybook and have also tested by running tooling command that ui packages changelogs are updated

## PR Checklist:

- [x] I have implemented all requirements? (see JIRA, project documentation).
- [x] I am not affecting someone else's work, If I am, they are included as a reviewer.
- [ ] I have added relevant tests (unit, integration or regression).
- [x] I have added comments to all the bits that are hard to follow.
- [x] I have added/updated Documentation.
- [x] I have updated the appropriate changelog with a line for my changes.

#### BEFORE
Screenshot 2024-08-14 at 15 19 19

#### AFTER
Screenshot 2024-08-14 at 15 18 40

### CODE REVIEW

**Review by tamara-causalens — APPROVED** [review]
Nice find!

**Review by krzysztof-causalens — APPROVED** [review]
